cfme/common/provider_views.py

```diff
@@ -248,5 +286,146 @@ def is_displayed(self):
 
     toolbar = View.nested(ProviderToolBar)
     sidebar = View.nested(ProviderSideBar)
-    entities = View.nested(ProviderEntities)
+    items = View.nested(ProviderItems)
     paginator = View.nested(PaginationPane)
+
+
+class BeforeFillMixin(object):
+    def before_fill(self):
+        if self.exists and not self.is_active():
+            self.select()
+
+
+class DefaultEndpointForm(View):
+    hostname = Input('default_hostname')
+    username = Input('default_userid')
+    password = Input('default_password')
+    confirm_password = Input('default_verify')
+
+    validate = Button('Validate')
+
+
+class SCVMMEndpointForm(DefaultEndpointForm):
```

Comment:
These provider specific views being in the common provider views module doesn't sit right with me.

I'd much rather see these views defined in their respective files in cfme/infrastructure/providers/.

Thoughts? I know it increases the import requirements there, but these aren't 'common' by definition it seems.